src/pages/Mypage/MainContent/BookMark/BookMark.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react';
+import BookMarkItem from './BookMarkItem/BookMarkItem';
+import './BookMark.scss';
+
+const BookMark = () => {
+  const [bookMarkList, setBookMarkList] = useState([]);
+
+  useEffect(() => {
+    fetch('data/BookMark/bookmarkData.json', {
```

Comment:
```suggestion
    fetch('/data/BookMark/bookmarkData.json', {
```
절대경로는 쓸거면 명확히 앞에 `/` 적어주세요! 아니라면 상대경로로 인식되어서 현재 위치한 url에 따라서 의도한대로 동작하지 않을수도 있습니다

Reply:
주의하겠습니다
모두 수정했습니다~